daemon/logger/fluentd/fluentd.go

```diff
@@ -216,6 +218,15 @@ func parseConfig(cfg map[string]string) (fluent.Config, error) {
 		}
 	}
 
+	asyncReconnectInterval := 0
+	if cfg[asyncReconnectIntervalKey] != "" {
+		ari64, err := strconv.ParseUint(cfg[asyncReconnectIntervalKey], 10, strconv.IntSize)
```

Comment:
Perhaps we should consider to parse the value as a `time.Duration` ([`time.ParseDuration()`](https://pkg.go.dev/time#ParseDuration)) to make it easier (and less error-prone) for the user to set the right interval. Having to specify the value in `ms` works "ok(ish)" if you want to, say, specify a second (`1000 ms`), but can become a bit awkward after that. It's also error-prone, for example, if a user wants to specify `1 second`, but specifies `1` and ends up with `1 ms`.

And looking at the code, I see we already use duration strings for `fluentd-retry-wait`, so it's a bit more consistent.

So I would suggest to; 

- use `time.ParseDuration()`
- validate the no negative values are specified
- perhaps also a minimum / maximum (not sure what reasonable values are? between 100ms and 10s ? (open to suggestions)

Would be good to have a minimal unit test (I see there's currently no tests in this package, which is a bit unfortunate, but let's not continue that "tradition" 😅)

Reply:
Thanks for the suggestion -- that sounds good to me. I'll have a look at your suggested changes on my fork and merge them in.

> Would be good to have a minimal unit test (I see there's currently no tests in this package, which is a bit unfortunate, but let's not continue that "tradition" 😅)

Yeah, I saw that and I guess lazily thought "ah, it's Christmas, time of tradition" 😉 -- I'll add some tests for this MR and probably cover the other test-cases in `parseConfig` while I'm at it 👍

Reply:
> MR and probably cover the other test-cases in parseConfig while I'm at it 

Thanks! Probably good to do that in a follow-up / separate PR to keep this PR "on topic"